cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/NetworkUtils.java

```diff
@@ -0,0 +1,78 @@
+/*
+*  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing,
+*  software distributed under the License is distributed on an
+*  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*  KIND, either express or implied.  See the License for the
+*  specific language governing permissions and limitations
+*  under the License.
+*/
+package org.ballerinalang.packerina;
+
+import org.ballerinalang.bre.Context;
+import org.ballerinalang.compiler.CompilerPhase;
+import org.ballerinalang.launcher.LauncherUtils;
+import org.ballerinalang.launcher.util.BCompileUtil;
+import org.ballerinalang.launcher.util.CompileResult;
+import org.ballerinalang.util.codegen.PackageInfo;
+import org.ballerinalang.util.codegen.ProgramFile;
+import org.ballerinalang.util.debugger.Debugger;
+import org.ballerinalang.util.program.BLangFunctions;
+
+import java.io.File;
+import java.nio.file.Path;
+
+import static org.ballerinalang.util.BLangConstants.USER_REPO_ARTIFACTS_DIRNAME;
+import static org.ballerinalang.util.BLangConstants.USER_REPO_SRC_DIRNAME;
+
+/**
+ * Util class for network calls
+ */
+public class NetworkUtils {
+    static CompileResult compileResult;
```

Comment:
This private right ?

Reply:
Included the CompileResult a private field with c8a9a47